Dockerfile

```diff
@@ -10,11 +10,15 @@ RUN npm install
 RUN npm run dev
 
 # Second stage from compatible php version
-FROM php:7.3
+FROM php:7.4
 
 # Copy container directory from previous stage
 COPY --from=0 /container .
 
+# Copy php-extension installer from Docker Hub image and install required php extensions
+COPY --from=mlocati/php-extension-installer /usr/bin/install-php-extensions /usr/bin
```

Comment:
Here's the Github for this extensions installer: [https://github.com/mlocati/docker-php-extension-installer](https://github.com/mlocati/docker-php-extension-installer)